### PR TITLE
opendkim - DKIM (Domain Keys Identified Mail) sender authentication s…

### PR DESCRIPTION
**Pull request by fritzkink**
…ystem v2.10.3

### CODE REVIEW

**Comment by AndWac** [comment]
This package depends on OpenSSL-1.0. Did you try to build it with OpenSSL-1.1?

File: components/mail/opendkim/Makefile [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+REQUIRED_PACKAGES += library/jansson
+REQUIRED_PACKAGES += library/libmemcached
+REQUIRED_PACKAGES += library/libmilter
+REQUIRED_PACKAGES += library/opendbx
+REQUIRED_PACKAGES += library/openldap
+REQUIRED_PACKAGES += library/security/openssl
+REQUIRED_PACKAGES += library/tre
+REQUIRED_PACKAGES += runtime/lua
+REQUIRED_PACKAGES += runtime/perl-534
+REQUIRED_PACKAGES += shell/ksh93
```
Comment (AndWac) [inline components/mail/opendkim/Makefile:105]: When I run "gmake REQUIRED_PACKAGES" on my system it'll replaces runtime/perl-534 with runtime/perl-524.

**Comment by fritzkink** [comment]
Your comment about openssl 1.1 made me rethink about it ;-). Initially I thought this package is to old for openssl 1.1 because it was released before openssl 1.1 was started. But I found some patches which made the build with openssl 1.1 successful.

File: components/mail/opendkim/Makefile (hunk 1 above)
Comment (fritzkink) [reply components/mail/opendkim/Makefile:105]: with new setup it is the same on my system.

**Pull request merged**